coderd/database/migrations/000256_workspace_agent_script_timings.up.sql

```diff
@@ -0,0 +1,19 @@
+ALTER TABLE workspace_agent_scripts ADD COLUMN id uuid unique not null default gen_random_uuid();
+ALTER TABLE workspace_agent_scripts ADD COLUMN display_name text not null default '';
```

Comment:
Is there anything else we can use here because an empty string for existing entries?

Reply:
I'm not really sure what we can do here. Definitely isn't ideal as already existing scripts will not show their name which is a bit silly.